Automattic/wp-calypso
# Jetpack Backup: Fix cloud Backup Badge

#### Proposed Changes

* Fixes an issue with the Backup Badge on sites without backups

#### Testing Instructions



* Create a new JN site and visit cloud.jetpack.com for it and ensure the pages load properly.
